integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -307,6 +324,277 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
+
+    final String auxiliaryImageVolumeName = "auxiliaryImageVolume1";
+    final String auxiliaryImagePath = "/auxiliary";
+
+    createSecretsForDomain(adminSecretName, encryptionSecretName, errorpathDomainNamespace);
+
+    // create stage dir for auxiliary image
+    Path errorpathAIPath3 = Paths.get(RESULTS_ROOT, "errorpathauxiimage3");
+    assertDoesNotThrow(() -> FileUtils.deleteDirectory(errorpathAIPath3.toFile()));
+    assertDoesNotThrow(() -> Files.createDirectories(errorpathAIPath3));
+
+    // create models dir and copy model for image
+    Path modelsPath3 = Paths.get(errorpathAIPath3.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath3));
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, "model.jms2.yaml"),
+        Paths.get(modelsPath3.toString(), "model.jms2.yaml"),
+        StandardCopyOption.REPLACE_EXISTING));
+
+    // unzip WDT installation file into work dir
+    unzipWDTInstallationFile(errorpathAIPath3.toString());
+
+    // create image1 with model and wdt installation files
+    createAuxiliaryImage(errorpathAIPath3.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), errorPathAuxiliaryImage3);
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", errorPathAuxiliaryImage3, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(errorPathAuxiliaryImage3),
+          String.format("docker push failed for image %s", errorPathAuxiliaryImage3));
+    }
+
+    // create domain custom resource using auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
+        domainUid, errorPathAuxiliaryImage3);
+    Domain domainCR = createDomainResource(domainUid, errorpathDomainNamespace,
+        WEBLOGIC_IMAGE_NAME + ":" + WEBLOGIC_IMAGE_TAG, adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, replicaCount, "cluster-1", auxiliaryImagePath,
+        auxiliaryImageVolumeName, errorPathAuxiliaryImage3);
+
+    // create domain and verify it is failed
+    logger.info("Creating domain {0} with auxiliary image {1} in namespace {2}",
+        domainUid, errorPathAuxiliaryImage3, errorpathDomainNamespace);
+    assertDoesNotThrow(() -> createDomainCustomResource(domainCR), "createDomainCustomResource throws Exception");
+
+    // check the domain event contains the expected error msg
+    String expectedErrorMsg =
+        "createDomain did not find the required domainInfo section in the model file /auxiliary/models/model.jms2.yaml";
+    verifyDomainEventContainsExpectedErrorMsg(timestamp, expectedErrorMsg);
+
+    // delete domain1
+    deleteDomainResource(errorpathDomainNamespace, domainUid);
+  }
+
+  /**
+   * Negative Test to create domain using a custom mount command that's guaranteed to fail, for example:  exit 1.
```

Comment:
Add the assertion we do here to check the warning.

Reply:
Can u describe the usecase, where we put the commabd "exit 1"

Reply:
This is the usecase for (g) Test a domain resource configuration using a custom common mount command that's guaranteed to fail, for example:  exit 1.